src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodLengthCheck.java

```diff
@@ -145,7 +145,8 @@ public void visitToken(DetailAST ast) {
                 openingBrace.findFirstToken(TokenTypes.RCURLY);
             final int length = getLengthOfBlock(openingBrace, closingBrace);
             if (length > max) {
-                log(ast, MSG_KEY, length, max);
+                final String methodName = ast.findFirstToken(TokenTypes.IDENT).getText();
```

Comment:
https://github.com/checkstyle/checkstyle/blob/f1e8346ef9dc13c9d778bb35a8821d43d409d003/src/main/java/com/puppycrawl/tools/checkstyle/DetailAstImpl.java#L358

It will work.